### PR TITLE
Create austrian-journal-of-development-studies.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/77489/style-request-austrian-journal-of-development-studies-jep

### CODE REVIEW

File: austrian-journal-of-development-studies.csl [hunk 1]
```diff
@@ -0,0 +1,310 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" name-delimiter="/" demote-non-dropping-particle="never" default-locale="de-AT">
```
Comment (rmzelle) [inline austrian-journal-of-development-studies.csl]: @POBrien333, if this is a polyglot CSL style it shouldn't have a default-locale, right?

Comment (rmzelle) [reply austrian-journal-of-development-studies.csl]: (or did you only develop/test this style for German?)

Comment (POBrien333) [reply austrian-journal-of-development-studies.csl]: thanks. An oversight. I've removed the default-locale.

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**